package.json

```diff
@@ -52,7 +52,7 @@
     "imagekit-javascript": "^3.0.2"
   },
   "engines": {
-    "node": ">=18.17.0"
+    "node": ">=12"
   },
   "peerDependencies": {
     "next": "^12 || ^13 || ^14",
```

Comment:
can you add next 15 here and test it out once?

Reply:
Done i have tested and made changes to make it compatible with next 15